src/lib.rs

```diff
@@ -86,6 +86,8 @@ pub const H: u32 = 1 << 31;
 
 /// Child index, whether hardened or not
 #[derive(Clone, Copy, Debug)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize), serde(into = "u32"))]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize), serde(from = "u32"))]
```

Comment:
```
#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize), serde(into = "u32", from = "u32"))]
```
?

Reply:
Too long, it'll be formatted awfully

Reply:
Formatted:
```
-#[cfg_attr(feature = "serde", derive(serde::Serialize), serde(into = "u32"))]
-#[cfg_attr(feature = "serde", derive(serde::Deserialize), serde(from = "u32"))]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(into = "u32", from = "u32")
+)]
```
I mean, I think I like it better this formatted way

Reply:
I don't, I like how it is now

Reply:
It even adds beauty as serialization-related stuff is in the first line, deserialization-related is in the second